### ISSUE TITLE
makedeb will not build packages on arm

### ISSUE THREAD

**Issue by gearhead** [hwittenborn/makedeb#263]
### Issue

Noticed that when installed on rpiOS on either armhf or arm64 that it would not build packages due to gcc flags issues. I compared the makepkg.conf file to those generated on arch ARM and was able to generate functional packages using these changes to the default `/etc/makepkg.conf` when running natively on the RPi:
armhf: (This is the standard RPiOS)
```
CARCH="armhf"
CHOST="armv6l-unknown-linux-gnueabihf"
...
CFLAGS="-march=armv6 -mfloat-abi=hard -mfpu=vfp -O2 -pipe -fstack-protector-strong -fno-plt -fexceptions \
        -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security \
        -fstack-clash-protection"
```

arm64: (this is the 64 bit version of RPiOS)
```
CARCH="arm64"
CHOST="aarch64-unknown-linux-gnu"
...
CFLAGS="-march=armv8-a -O2 -pipe -fstack-protector-strong -fno-plt -fexceptions \
        -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security \
        -fstack-clash-protection"
```

### Installation source

APT Repository

### Installed release type

Alpha

### Installed makedeb version

makedeb 6.0.0-alpha9

**Comment by archanox** [comment]
I've recently discovered that this issue may be extending to RISC-V too, thanks to the helpful folk over [here](https://users.rust-lang.org/t/rustup-risc-v-install-error/116846?u=archanox) on the rust forums.

Tl;dr the error I'm getting is
```

The following warnings were emitted during compilation:

warning: cc: error: ‘-march=x86-64’: ISA string must begin with rv32 or rv64

error: failed to run custom build command for `sys-info v0.9.1`

Caused by:
  process didn't exit successfully: `/home/debian/rustup/src/rustup-1.25.2/target/release/build/sys-info-1a75f39fa2e70555/build-script-build` (exit status: 1)
  --- stdout
  TARGET = Some("riscv64gc-unknown-linux-gnu")
  OPT_LEVEL = Some("3")
  HOST = Some("riscv64gc-unknown-linux-gnu")
  CC_riscv64gc-unknown-linux-gnu = None
  CC_riscv64gc_unknown_linux_gnu = None
  HOST_CC = None
  CC = None
  CFLAGS_riscv64gc-unknown-linux-gnu = None
  CFLAGS_riscv64gc_unknown_linux_gnu = None
  HOST_CFLAGS = None
  CFLAGS = Some("-march=x86-64 -mtune=generic -O2 -pipe -fno-plt -fexceptions         -Wp,-D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security         -fstack-clash-protection -fcf-protection")
  CRATE_CC_NO_DEFAULTS = None
  DEBUG = Some("false")
  CARGO_CFG_TARGET_FEATURE = None
  running: "cc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-march=rv64gc" "-mabi=lp64d" "-mcmodel=medany" "-march=x86-64" "-mtune=generic" "-O2" "-pipe" "-fno-plt" "-fexceptions" "-Wp,-D_FORTIFY_SOURCE=2" "-Wformat" "-Werror=format-security" "-fstack-clash-protection" "-fcf-protection" "-o" "/home/debian/rustup/src/rustup-1.25.2/target/release/build/sys-info-73bd6f2e2f8646f5/out/c/linux.o" "-c" "c/linux.c"
  cargo:warning=cc: error: ‘-march=x86-64’: ISA string must begin with rv32 or rv64
  exit status: 1

  --- stderr


  error occurred: Command "cc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-march=rv64gc" "-mabi=lp64d" "-mcmodel=medany" "-march=x86-64" "-mtune=generic" "-O2" "-pipe" "-fno-plt" "-fexceptions" "-Wp,-D_FORTIFY_SOURCE=2" "-Wformat" "-Werror=format-security" "-fstack-clash-protection" "-fcf-protection" "-o" "/home/debian/rustup/src/rustup-1.25.2/target/release/build/sys-info-73bd6f2e2f8646f5/out/c/linux.o" "-c" "c/linux.c" with args "cc" did not execute successfully (status code exit status: 1).
```

I'm not 100% sure that this issue is 1 for 1, but it's essentially the same core problem of makedeb not supporting any architecture other than x86.
